### ISSUE TITLE
Skins: Add safe polygon to the popovers

### ISSUE THREAD

**Issue by sampotts** [videojs/v10#774]
Currently the popovers can disappear if you're slow with moving the cursor. That's because we're missing something like [`safePolygon`](https://floating-ui.com/docs/usehover#safepolygon).
